README.md

```diff
@@ -83,7 +86,13 @@ git clone https://github.com/nginxinc/docker-nginx-controller.git
 ```
 
 ```bash
-cd docker-nginx-controller/<os>
```

Comment:
A new section should be added to the readme for the NAP container.
Instead of extending the existing section - Building a NAP enabled container.